PED/SetPedComponentVariation.md

```diff
@@ -8,33 +8,31 @@ ns: PED
 void SET_PED_COMPONENT_VARIATION(Ped ped, int componentId, int drawableId, int textureId, int paletteId);
 ```
 
-```
-paletteId/palletColor -  0 to 3.   
-enum PedVariationData  
-{  
-	PED_VARIATION_FACE = 0,  
-	PED_VARIATION_HEAD = 1,  
-	PED_VARIATION_HAIR = 2,  
-	PED_VARIATION_TORSO = 3,  
-	PED_VARIATION_LEGS = 4,  
-	PED_VARIATION_HANDS = 5,  
-	PED_VARIATION_FEET = 6,  
-	PED_VARIATION_EYES = 7,  
-	PED_VARIATION_ACCESSORIES = 8,  
-	PED_VARIATION_TASKS = 9,  
-	PED_VARIATION_TEXTURES = 10,  
-	PED_VARIATION_TORSO2 = 11  
-};  
-Usage:   
-SET_PED_COMPONENT_VARIATION(playerPed, PED_VARIATION_FACE, GET_NUMBER_OF_PED_DRAWABLE_VARIATIONS(playerPed, PED_VARIATION_FACE), GET_NUMBER_OF_PED_TEXTURE_VARIATIONS(playerPed, PED_VARIATION_FACE, 0), 2);  
-List of component/props ID  
-gtaxscripting.blogspot.com/2016/04/gta-v-peds-component-and-props.html  
-```
+This native is used to set component variation on a ped. Components, drawables and textures IDs are related to the ped model. 
 
 ## Parameters
-* **ped**: 
-* **componentId**: 
-* **drawableId**: 
-* **textureId**: 
-* **paletteId**: 
+* **ped**: The ped handle.
+* **componentId**: The component that you want to set.
+* **drawableId**: The drawable id that is going to be set.
+* **textureId**: The texture id of the drawable.
+* **paletteId**: 0 to 3.
+
+## MP Freemode list of components
```

Comment:
All this (and below) must be moved above `## Parameters`. This is still a requirement for any 'description' text to be detected. Any description text MUST be placed directly below the C code block and ABOVE any other sections like `## Examples` `## Parameters` `## Return value` and similar.

Currently all this text will not end up in the description section of the docs.